tests/test_responder.py

```diff
@@ -830,6 +831,9 @@ def create_asset(static_dir, name=None, parent_dir=None):
 
 @pytest.mark.parametrize("static_route", [None, "/static", "/custom/static/route"])
 def test_staticfiles(tmpdir, static_route):
+    if sys.platform == "win32" and static_route is None:
+        raise pytest.skip("Route 'None' not supported on Windows")
+
```

Comment:
We may want to convert this note into a dedicated ticket, then even process it as a bug report?

Reply:
Thanks for your valuable suggestions below. I've deferred this issue to a dedicated ticket.

- GH-548